src/main.js

```diff
@@ -47,13 +40,6 @@ parser.addArgument(
         choices: ['silly', 'debug', 'verbose', 'info', 'warn', 'error'],
         help: "What level to log at.",
     });
-parser.addArgument(
-    ['--likeprod'],
-    {
-        action: 'storeTrue',
-        help: "Only useful with --dev. Tells the dev server to cache and timeout like production would",
```

Comment:
I guess the timeout portion already changed?

Reply:
Yeah, we just rely on the request from fastly timing out now. It was a hangover from when the render server had to respond quickly.